CONTRIBUTING.md

```diff
@@ -157,6 +159,15 @@ Hold my beer, I got this
 
 
 
+## World Air Quality Index's API
+
+Ozone uses the World Air Quality Index's API under the hood to fetch its data. We have included some links to information about the API to help during contribution.
+
+* Find information on the World Air Quality Index website [here](https://waqi.info/)
+* Find information on API used by Air Quality Programs [here](https://aqicn.org/api/) 
+
```

Comment:
@Tomiiwa Great work! i have one small thing i would like for you to add

this [link ](https://aqicn.org/api/) is great and it contains all APIs used by ozone , and itself contains reference to the JSON API among other APIs 

can u add a link to the JSON API below it?

Reply:
alright, will do

Reply:
and if you could change the description of the second link to reflect the idea that it offers multiple APIs and not one and thank u very much :)